### DISCUSSION TITLE
how to mount  usb otg-hs on STM32F429I-DiISC1

### DISCUSSION THREAD

**Discussion by Sujung84** [zephyrproject-rtos/zephyr#36923] [Q&A]
Hello.
I develop STM32F429I-DiISC1 with Zephyr.
I tried  to register usb otg-hs using samples/subsys/usb/mass/, but it is failed. 
In samples/subsys/usb/mass/Kconfig, if DISK_DRIVER_FLASH is true , DISK_DRIVER_FLASH name is NAND
config MASS_STORAGE_DISK_NAME
     default "NAND" if DISK_DRIVER_FLASH
      default "RAM" if DISK_DRIVER_RAM
      default "SD" if DISK_DRIVER_SDMMC

Shouldn't there be a code to register as a flash device when USB OTG is included in MASS STORAGE DISK NAME or when USB OTG is recognized?

please inform to me how to mount usb otg hs.

**Comment by jfischer-no** [comment]
I do not understand the question. Maybe it helps to say that there is no USB host or OTG support in Zephyr.
